mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -477,8 +492,27 @@ resource "google_dns_record_set" "foobar" {
 `, name, name, name, ttl)
 }
 
-func testAccDnsRecordSet_routingPolicyWRR(zoneName, addr2 string, ttl int, weight float64) string {
+func testAccDnsRecordSet_routingPolicyWRR(networkName, backendName, forwardingRuleName, zoneName string, ttl int) string {
 	return fmt.Sprintf(`
+resource "google_compute_network" "default" {
```

Comment:
These test scenarios are loosely based off of https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_forwarding_rule and a set of internal tests that is linked in b/239887327.